irs-api/pom.xml

```diff
@@ -170,16 +161,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <!-- Update jsoup manually to avoid a vulnerability; can be removed after jsonschemafriend updates their dependency -->
```

Comment:
No, there is no new version for this dependency and no need to update the dependency since we could resolve the dependency vulnerability by updating the library manually as stated in the comment. 
This is just a reminder for someone who touches this dependency to check.